app/Filament/Resources/Media/RelationManagers/EventsRelationManager.php

```diff
@@ -29,15 +31,22 @@ public function form(Schema $schema): Schema
                 Select::make('media_event_type_id')
                     ->relationship(name: 'mediaEventType', titleAttribute: 'name')
                     ->getOptionLabelFromRecordUsing(fn ($record) => $record->name->value)
-                    ->required(),
+                    ->required()
+                    ->live(),
                 DatePicker::make('occurred_at')
                     ->label('Date')
                     ->required(),
                 Textarea::make('comment')
-                    ->columnSpanFull(),
+                    ->columnSpanFull()
+                    ->required(fn (Get $get): bool => $get('media_event_type_id') == self::commentEventTypeId()),
             ]);
     }
 
+    private static function commentEventTypeId(): ?int
+    {
+        return once(fn () => MediaEventType::where('name', MediaEventTypeName::COMMENT)->first()?->id);
+    }
+
```

Comment:
Is it worth refactoring this into some kind of utility? 

Could we cache all these IDs inside of the MediaEventType model or enum? 

That way, we can generalize the solution for all media event type names, and we don't need to repeat this similar code in MediaEventResource.php